tools/mkdocs_macros_bidsschema/main.py

```diff
@@ -0,0 +1,44 @@
+"""This package is used to build elements in the bids-specification schema into
+MarkDown format for the specification text.
+
+Functions decorated in "define_env()" are callable throughout the
+specification and are run/rendered with the mkdocs plugin "macros".
+"""
+import os
+import sys
+
+schemacode_path = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
```

Comment:
Sorry @yarikoptic, but this is the current version of what you commented on in https://github.com/bids-standard/bids-specification/pull/610#discussion_r528005803. Should I add `__init__.py` to this line?